src/apis/order_api.ts

```diff
@@ -64,6 +76,10 @@ export const OrderAPIErrors = {
 
     INVALID_UNDERWRITER_SIGNATURE: () =>
         singleLineString`Underwriter signature is not valid for debt order`,
+
+    ADAPTER_DOES_NOT_CONFORM_TO_INTERFACE: () =>
+        singleLineString`Supplied adapter does not conform to the required
```

Comment:
The world `required` is possibly awkward here and could probably be removed. "The supplied adapter does not conform to the base adapter interface."